kore/test/Test/Kore/Builtin.hs

```diff
@@ -78,44 +89,50 @@ test_internalize =
 
     , internalizes  "unitMap"
         unitMap
-        (mkMap [])
+        (mkMap [] True False False)
```

Comment:
I think these tests would be easier to read if instead of using `Bool` values we'd create some datatypes with more suggestive names.

Reply:
I experimented with different data types but I think that my latest changes strike the best balance between readability and complexity/overhead. What I ended up doing was just to define some constants to be `True` and `False`  and I'm using those constants in the tests instead of the naked bools. I hope this addresses your point.